### PR TITLE
apache-spark 1.6.2

### PR DESCRIPTION
**Pull request by fredkelly**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/apache-spark.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class ApacheSpark < Formula
   desc "Engine for large-scale data processing"
   homepage "https://spark.apache.org/"
-  url "https://www.apache.org/dyn/closer.lua?path=spark/spark-1.6.1/spark-1.6.1-bin-hadoop2.6.tgz"
-  version "1.6.1"
-  sha256 "09f3b50676abc9b3d1895773d18976953ee76945afa72fa57e6473ce4e215970"
+  url "https://www.apache.org/dist/spark/spark-1.6.2/spark-1.6.2-bin-hadoop2.6.tgz"
```
Comment (DomT4) [inline Formula/apache-spark.rb]: Why the move away from `dyn` here?

**Comment by fredkelly** [comment]
No reason, I hadn't realised there was a benefit in the `/dyn/` URL - I will amend.

**Comment by DomT4** [comment]
It's a geolocating mirror, similar to GNU's `ftpmirror` setup. Just means people get slightly quicker downloads & we don't pound Apache's central servers.

**Comment by fredkelly** [comment]
@DomT4 thanks for the tip, I've updated my branch.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/9ee1f62fcf5758058413f7b6cd1e6e8572562caa. Thank you for your contribution to Homebrew @fredkelly; we appreciate it! :smiley_cat:

**Pull request merged**